### PR TITLE
Bump dependencies to Kubernetes 1.20

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
Steps:
```
go get -u ./...
./release-tools/go-get-kubernetes 1.20.0
# clean up versions in go.mod
GO111MODULE=on go mod tidy
GO111MODULE=on go mod vendor
```

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,49 +3,69 @@ module github.com/kubernetes-csi/external-snapshotter/v4
...
+	golang.org/x/sys v0.0.0-20201214095126-aec9a390925b // indirect
+	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf // indirect
+	golang.org/x/time v0.0.0-20201208040808-7e3f01d25324 // indirect
+	google.golang.org/appengine v1.6.7 // indirect
+	google.golang.org/genproto v0.0.0-20201211151036-40ec1c210f7a // indirect
+	google.golang.org/grpc v1.34.0
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/api v0.20.0
+	k8s.io/apimachinery v0.20.0
+	k8s.io/client-go v1.5.1
```
Comment (jsafrane) [inline go.mod:35]: This is suspicious, I would expect `v0.20.0`

Comment (xing-yang) [reply go.mod:35]: Yeah, this looks weird.  It's picked up automatically and it is a valid release: https://github.com/kubernetes/client-go/releases/tag/v1.5.1

Comment (xing-yang) [reply go.mod]: Fixed.

**Pull request merged**